src/libs/SearchQueryUtils.ts

```diff
@@ -319,6 +320,11 @@ function buildSearchQueryJSON(query: SearchQueryString) {
         result.hash = primaryHash;
         result.recentSearchHash = recentSearchHash;
 
+        if (result.policyID && !Array.isArray(result.policyID)) {
```

Comment:
NAB but maybe we should check that `result.policyID` is a string in this case, since any other case would be invalid